fury/data/fetcher.py

```diff
@@ -370,6 +379,48 @@ def fetcher():
     )
 
 
+def read_viz_cubemap(name, suffix_type=1, ext='.jpg'):
+    """Read specific cube map with specific suffix type and extension.
+
+    Parameters
+    ----------
+    name : str
+    suffix_type : int, optional
+        0 for numeric suffix (e.g., skybox_0.jpg, skybox_1.jpg, etc.), 1 for
+        -p/nC encoding where C is either x, y or z (e.g., skybox-px.jpeg,
+        skybox-ny.jpeg, etc.), 2 for pos/negC where C is either x, y, z (e.g.,
+        skybox_posx.png, skybox_negy.png, etc.), and 3 for position in the cube
+        map (e.g., skybox_right.jpg, skybox_front.jpg, etc).
+    ext : str, optional
+        Image type extension. (.jpg, .jpeg, .png, etc.).
```

Comment:
Not easy, might be simpler to just list all the names. To discuss

Reply:
Agree. The important thing is the order in which the images are added to the cube map texture.